Separate "Privacy, search, and services" into "Privacy and security" & "Search and services" (#3127)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

Reviewer: there are 35 hits on edge://settings/privacy in .md file.  That url currently brings up the old Settings > "Privacy, search, and services** UI:

![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/44aca917-3ccf-4acb-ad0a-0105d34af2bb)

Do we need to check and change all 35 spots?
@captainbrosset @apryzgoda

Reviewer: line 165:
go to `edge://settings/privacy`, scroll down to the **Services** section

Reviewer 2: > there are 35 hits on edge://settings/privacy in .md file. That url currently brings up the old Settings > "Privacy, search, and services** UI:

> Do we need to check all 35 spots, and change some? Has this been checked? I recommend deleting that special URL and replacing by GUI nav steps/path (standard way of doc'ing UI), which will help maintainers & users. 
> 

I agree. We should check all 35 hits and use the GUI nav.

Reviewer: closing, since 35 spot-comments were entered, and have so far resolved all but 8 of them, using boilerplate:
```
select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**, and then 
```
```
select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, and then 
```

Reviewer: line 16:
This document provides links to go to Microsoft Edge settings and other pages.  The shortcuts URLs start with `edge://` such as `edge://favorites` and `edge://settings/privacy`.  To go to the pages, type the bolded text directly into the Microsoft Edge address bar.  The pages are only viewable in Microsoft Edge.

At some point, recommend deleting this paragraph and approach.  Give standard GUI nav path instead.

Reviewer: probably keep line 16 paragraph as-is in this PR, and remove in a later PR.
This PR starts moving away from special URLs to standard GUI nav path format.

Reviewer: Spots (tracking-prevention.md) to assess in the regular docs in the repo:
https://learn.microsoft.com/microsoft-edge/web-platform/tracking-prevention#levels-of-tracking-prevention - 
* "Microsoft Edge currently offers users three levels of tracking prevention, which are selected by navigating to edge://settings/privacy." 
* "Go to `edge://settings/privacy`."
* "This option is accessed via the page info flyout or through the `edge://settings/privacy/trackingPreventionExceptions` page."

Reviewer: replaced by spot-comments

Reviewer: resolving b/c replaced this comment by a spot-comment

If your search provider is Bing, a resettable identifier unique to your browser is sent with the data. It helps Bing understand the search query and query session. Other autosuggest service identifiers are sent to your default search engine to complete the search suggestions. Your IP address and cookies are sent to your default search provider to increase the relevance of the search results.

> When you select the address bar, a signal is sent to your default search provider. The signal tells the provider to prepare suggestions.  The typed characters and search queries aren't sent to Microsoft unless your search provider is Bing.  To enable sending data to your default search provider, go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  In **Services**, select **Address bar** and turn on the **Show me search and site suggestions using my typed characters** setting.  If you turn off the setting, your typed characters are no longer sent to your default search provider.  Your search queries are still sent to your default search provider to provide search results.

Reviewer: ```
To enable sending data to your default search provider, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Search and services**, and then select **Address bar** and turn on 
the **Show me search and site suggestions using my typed characters** setting.
```

<!-- The Privacy settings have been updated from what is described here -->

If Microsoft Edge detects typing in the address bar that might contain sensitive information, it does not send the typed text. Sensitive information includes authentication credentials, local file names, or URL data that is normally encrypted.

> You can configure Microsoft Edge to collect diagnostic data about the address bar. Collected data includes the number of offered queries for all search providers. Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  In **Personalize your web experience** turn on the **Improve your web experience by allowing Microsoft to use your browsing history from this account for personalizing advertising, search, news, and other Microsoft services** setting.

Reviewer: ```
Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** 
> **Search and services**, and then in **Personalize your web experience**, 
turn on the setting **Improve your web experience by allowing Microsoft to use 
your browsing history from this account for personalizing advertising, search, news, 
and other Microsoft services**.
```

You can control whether typed characters are sent to your default search provider to provide search and site suggestions.  To enable or disable sending your typed characters in the Address bar to your default search provider:

> 1. Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**.<!-- todo: update as separate groups of settings --><!--edge://settings/privacy-->

Reviewer: did the toggle **Show me search and site suggestions using my typed characters** end up in "Privacy and security" or "Search and services"?
@PatrizioBytes

Reviewer: pls help integrate these incoming steps from the rewrite of the Address Bar h2 section that's in "main" branch, into this PR's scenario of separating the "Privacy, search, and services" settings into "Privacy and security" & "Search and services".

<!-- ------------------------------ -->
#### Enabling collecting Address bar diagnostic data

> You can configure Microsoft Edge to collect diagnostic data about the Address bar.  Collected data includes the number of offered queries for all search providers.<!-- todo: is the number of offered queries the only data that's collected through diagnostic data?  if not, shouldn't list that one only.  specify for Enterprise Search Provider & specify for all providers -->  To enable collecting diagnostic data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**,<!-- todo: update as separate groups of settings --> and then in the **Personalization & advertising** section, turn on the setting **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news.**

Reviewer: did the toggle **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news** end up in "Privacy and security" or "Search and services"?
@PatrizioBytes

<!-- ------------------------------ -->
#### Deleting data

> Typed characters and the websites that you visit are stored locally on the device, per profile.  To delete that data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**,<!-- todo: update as separate groups of settings --> and then in the **Delete browsing data** section, click the **Choose what to clear** button; then in the **Delete browsing data** dialog, click the **Clear now** button.

Reviewer: did this UI end up in "Privacy and security" or "Search and services"? --
in the **Delete browsing data** section, click the **Choose what to clear** button; then in the **Delete browsing data** dialog, click the **Clear now** button.
@PatrizioBytes

If Bing is your default search provider and you are signed in,<!-- todo: reformat paragraph as 3 list items starting here? --> you can delete your searches by using the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).

> To clear your browsing history and delete websites from appearing as suggestions in the Address bar: select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**,<!-- todo: update as separate groups of settings --> and then in the **Delete browsing data** section, click the **Choose what to clear** button; then in the **Delete browsing data** dialog, click the **Clear now** button.

Reviewer: 2nd instance:
did this UI end up in "Privacy and security" or "Search and services"? --
in the **Delete browsing data** section, click the **Choose what to clear** button; then in the **Delete browsing data** dialog, click the **Clear now** button.
@PatrizioBytes

> The user can modify this permission anytime by going to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle.

Reviewer 2: ```suggestion
The user can modify this permission anytime by selecting **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle.
```

Reviewer: ```
To modify this permission, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot**, and then use the toggle **Allow access to any webpage or PDF**.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may

When you sign in to Microsoft Edge with a personal Microsoft Account (MSA), your collections, saved items, notes, and images are automatically saved on Microsoft cloud for free. You can access your collections from Microsoft Edge on any device where you're signed in. You can even view and manage your collections online in any browser by visiting the [Collections portal](https://www.bing.com/saves) at Bing.com. Collections also enhance your experience with a personalized interest feed, if your personalization settings support this feature.

> If you want to disable Collections for a signed-in profile using a personal account, go to `edge://settings/privacy`,<!--todo: check, maybe replace by GUI nav path--> scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.

Reviewer: ```
If you want to disable Collections for a signed-in profile using a personal account, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Search and services**, and then turn off the toggle for 
**Show Collections and follow content creators in Microsoft Edge**.
```

You can clear the crash diagnostic data stored locally on a device. Go to `edge://crashes` and select the **Clear all** button.

> To turn off crash diagnostic data collection on Windows 10 and later, open **Start** > **Settings** > **Privacy** and select **Diagnostics & feedback**.  For versions of Microsoft Edge on all other platforms, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. You can manage diagnostic data collection at the enterprise level. See [group policies managed by your organization](/deployedge/microsoft-edge-enterprise-privacy-settings).

Reviewer: ```
For versions of Microsoft Edge on all other platforms, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then turn off the setting 
**Help improve Microsoft products by sending optional diagnostic data about 
how you use the browser, websites you visit, and crash reports**.
```

The Microsoft Edge team respects the sensitivity of diagnostic data by restricting data access or removing personal information. To reset the identifier unique to your browser on Windows 10 and later devices, select **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**, and then select **Delete** under **Delete diagnostic data**, or change your setting under **Diagnostic data** from **Full** to **Basic** or turn off **Optional diagnostic data**.

> On other platforms, to generate a new resettable identifier (ID) that is unique to your browser, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. The reset (ID) functionality may be different for devices managed with group policies set by your organization.

Reviewer: ```
On other platforms, to generate a new resettable identifier (ID) that is unique to your browser, 
select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then turn off the setting 
**Help improve Microsoft products by sending optional diagnostic data about 
how you use the browser, websites you visit, and crash reports**.
```

To help us improve Microsoft products and services, diagnostic data is aggregated, with personal identifiers removed, and stored for up to two years. Because the diagnostic data is not collected from or stored with your Microsoft account, the diagnostic data may not be viewed or deleted from your [Microsoft privacy dashboard](https://account.microsoft.com/privacy/). To delete the diagnostic data on Windows 10 and later, select **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**, and then select **Delete** under **Delete diagnostic data**. The delete diagnostic data functionality is only supported on Windows 10 version 1803 or later. For more information, see [Diagnostics, feedback, and privacy in Windows 10 and later](https://support.microsoft.com/help/4468236).

> For Microsoft Edge on Windows 10 and later, sending Optional diagnostic data is determined by your Windows diagnostic data setting. The setting is reflected in `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  Change the Windows settings by going to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**. On all other platforms, to control the collection of diagnostic data, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and turn on or off **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports**. The setting is used for all profiles associated with the installation of Microsoft Edge on your device.  The setting is not synced across devices. The setting applies to **InPrivate** browsing and Guest mode. Information about websites you visit is never sent while browsing **InPrivate** or in **Guest** mode. If your device is managed with group policies set by your organization, it is described in `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->

Reviewer: ```
The setting is reflected in **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, but is not set through that location.  
Instead, change the Windows settings by going to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**.
On platforms other than Windows, to control the collection of diagnostic data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then use the toggle 
**Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports**.
...
If your device is managed by using group policies that are set by your organization, the setting is reflected in **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**.
```
Had to do some rewriting of paragraph to move it to clearer format, to enable maintaining the paragraph.
Also, paragraph is far too long to be comprehensible, and must be broken up -- along w/ adding h4 headings -- but that's out of scope for this PR.  
@PatrizioBytes @apryzgoda

> For Microsoft Edge on Windows 10 and later, sending Optional diagnostic data is determined by your Windows diagnostic data setting. The setting is reflected in `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  Change the Windows settings by going to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**. On all other platforms, to control the collection of diagnostic data, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and turn on or off **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports**. The setting is used for all profiles associated with the installation of Microsoft Edge on your device.  The setting is not synced across devices. The setting applies to **InPrivate** browsing and Guest mode. Information about websites you visit is never sent while browsing **InPrivate** or in **Guest** mode. If your device is managed with group policies set by your organization, it is described in `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->

Reviewer: h2 section requires h4 subheadings.  out of scope.  so merely added comment:
```
<!-- todo: break up paragraph, add h4 headings in this h2 section -->
```

> As of March 6, 2024, Microsoft Edge diagnostic data is collected separately from Windows diagnostic data on Windows 10 (version 22H2 and newer) and Windows 11 (version 23H2 and newer) devices in the European Economic Area. For these versions of Windows, the diagnostic data setting for Microsoft Edge is controlled through `edge://settings/privacy`,<!--todo: check, maybe replace by GUI nav path--> rather than being a Windows setting that's reflected there.

Reviewer: ```
For these versions of Windows, the diagnostic data setting for Microsoft Edge is controlled through **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**, rather than being a Windows setting that's reflected there.
```

Microsoft Edge supports Widevine by Google DRM and the option is on by default.  Microsoft Edge periodically fetches updates for Widevine from Google servers.  The use of Widevine may include communications to Google.  To opt-out of using Widevine in Microsoft Edge, go to `edge://flags/#edge-widevine-drm` and turn off the Widevine DRM setting.  Widevine has the capability to create a unique device identifier and transmit it to Google.  For more specific information on Widevine and privacy, see the Google privacy policy.

> Microsoft Edge supports the Flash Access DRM by Adobe, which is used by some sites instead of HTML5. You are prompted to allow Adobe Flash when a site requests it.  When a site uses the Flash Access DRM by Adobe, Microsoft Edge gives Adobe access to a unique device identifier.  You can clear and reset any locally stored instances of the identifier. Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  In **Clear browsing data**. select **Choose what to clear**, select the checkbox for **Cookies and other site data**, and select **Clear now** to remove any stored identifiers. To stop Adobe Flash DRM from ever being used, go to `edge://settings/content/flash`.

Reviewer: ```suggestion
Microsoft Edge supports the Flash Access DRM by Adobe, which is used by some sites instead of HTML5. You are prompted to allow Adobe Flash when a site requests it.  When a site uses the Flash Access DRM by Adobe, Microsoft Edge gives Adobe access to a unique device identifier.  You can clear and reset any locally stored instances of the identifier. Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  In **Clear browsing data**, select **Choose what to clear**, select the checkbox for **Cookies and other site data**, and select **Clear now** to remove any stored identifiers. To stop Adobe Flash DRM from ever being used, go to `edge://settings/content/flash`.
```
change period to comma

> Microsoft Edge supports the Flash Access DRM by Adobe, which is used by some sites instead of HTML5. You are prompted to allow Adobe Flash when a site requests it.  When a site uses the Flash Access DRM by Adobe, Microsoft Edge gives Adobe access to a unique device identifier.  You can clear and reset any locally stored instances of the identifier. Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  In **Clear browsing data**. select **Choose what to clear**, select the checkbox for **Cookies and other site data**, and select **Clear now** to remove any stored identifiers. To stop Adobe Flash DRM from ever being used, go to `edge://settings/content/flash`.

Reviewer: ```
To clear and reset any locally stored instances of the identifier, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**.  Then in **Clear browsing data**, 
select **Choose what to clear**, select the checkbox for **Cookies and other site data**, and select **Clear now**.
```

<!-- ====================================================================== -->
## Do Not Track

> You can enable **Do Not Track** on Microsoft Edge. Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  Turn on the **Send "Do Not Track" requests** setting.  If you enable the **Do Not Track** feature, Microsoft Edge sends a `DNT:1 HTTP` header with your outgoing HTTP, HTTPS, and SPDY browsing traffic requests. This feature tells websites you visit not to use trackers. However, enabling the **Send "Do Not Track" requests** setting does not guarantee that the websites aren't able to track you. Some sites may honor the request by showing you ads not based on any previous browsing. Microsoft Edge does not control whether or not the request is honored. You can help prevent websites from tracking you. Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  Change the **Tracking prevention** setting to **Balanced** or **Strict**.

Reviewer: ```
To help prevent websites from tracking you, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then change the **Tracking prevention** setting 
to **Balanced** or **Strict**.
```

When you use **Guest** mode, Microsoft Edge does not send **Do Not Track** requests.  When you use **InPrivate** browsing, Microsoft Edge only sends **Do Not Track** requests if the **Send "Do Not Track" requests** setting is turned on for the profile you are using.

<!-- ====================================================================== -->
## Downloads

> Microsoft Edge lets you download files safely and securely.  To choose where files are downloaded on your device, go to `edge://settings/downloads`.  If SmartScreen is enabled, information about your file, such as the file name and URL, are sent to SmartScreen to check the reputation of the file. The reputation check helps you avoid accidentally downloading known malware that is known to hurt your device.  To change SmartScreen settings, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and use the **Microsoft Defender SmartScreen** toggle. For more information about SmartScreen, see the [SmartScreen](#smartscreen) section.

Reviewer 3: correct.

Reviewer: ```
To change SmartScreen settings, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then use the **Microsoft Defender SmartScreen** toggle.
```

<!-- ====================================================================== -->
## Do Not Track

> You can enable **Do Not Track** on Microsoft Edge. Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  Turn on the **Send "Do Not Track" requests** setting.  If you enable the **Do Not Track** feature, Microsoft Edge sends a `DNT:1 HTTP` header with your outgoing HTTP, HTTPS, and SPDY browsing traffic requests. This feature tells websites you visit not to use trackers. However, enabling the **Send "Do Not Track" requests** setting does not guarantee that the websites aren't able to track you. Some sites may honor the request by showing you ads not based on any previous browsing. Microsoft Edge does not control whether or not the request is honored.  To help prevent websites from tracking you, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**, and then change the **Tracking prevention** setting to **Balanced** or **Strict**.

Author: Correct, edge://settings/privacy still contains the Do Not Track option after the settings split change.
I will push a commit to match the other GUI nav instructions.

> You can enable **Do Not Track** on Microsoft Edge. Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  Turn on the **Send "Do Not Track" requests** setting.  If you enable the **Do Not Track** feature, Microsoft Edge sends a `DNT:1 HTTP` header with your outgoing HTTP, HTTPS, and SPDY browsing traffic requests. This feature tells websites you visit not to use trackers. However, enabling the **Send "Do Not Track" requests** setting does not guarantee that the websites aren't able to track you. Some sites may honor the request by showing you ads not based on any previous browsing. Microsoft Edge does not control whether or not the request is honored.  To help prevent websites from tracking you, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**, and then change the **Tracking prevention** setting to **Balanced** or **Strict**.

Author: Fixed it in the last commit.

<!-- ====================================================================== -->
## Find on page 

> Use the "Find on page" feature to search a webpage for key words.  In a webpage, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  Or, click **Settings and more** (**...**), and then select **Find on page**.  A search box opens.  Enter a word, phrase, or other string into this search box.  As you type, all occurrences of the string in the page are highlighted. You can then move through the search results by clicking the **Previous result** and **Next result** buttons.

Reviewer 3: this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reviewer: The **Find on page** command is directly on the **Settings and more** menu, as the existing text above states.
The end of this section has been reworded as gui nav path:
```
To stop Microsoft Edge from finding related matches in **Find on page**, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Search and services**, and then turn off the toggle **Include related matches in Find on page**.
```

To stop Microsoft Edge from finding related matches in **Find on page**:

> 1. Select **Settings and more** (**...**) > **Settings** > **Search and services**, or, go to `edge://settings/searchServices`.

Reviewer: ```suggestion
1. Select **Settings and more** (**...**) > **Settings** > **Search and services**.  Or, go to `edge://settings/searchServices`.
```
two separate sentences enables the first sentence/step/action to be short & simple.  
i want to move entire article toward the GUI approach, and away from the "go to special url" approach.  The GUI approach is self-contained and sufficient, should keep separate from the esoteric "go to special url" approach.

Reviewer: general design conventions in article: I doubt that the article should ever state the special url, such as "Go to edge://settings/searchServices".  Seems like noise & nonstandard instructions, detracting from the recommended, GUI nav approach.  Can we just delete:
 (Or, go to `edge://settings/searchServices`.)
@apryzgoda - not sure why the article ever gives these esoteric URLs.  GUI nav approach is standard and sufficient.

Reviewer: ```
To stop Microsoft Edge from finding related matches in **Find on page**, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, and then turn off the toggle **Include related matches in Find on page**.
```

History supports views to display images of your web navigations and enhance your history experience.  All history data is stored locally on the device, and is organized per Microsoft Edge profile.  Images are saved for the duration the history exists (90 days).

> If you want Microsoft Edge not to capture images for your history, select the **Turn off images** button in the history hub.  Or, go to `edge://settings/privacy`,<!--todo: check, maybe replace by GUI nav path--> scroll down to the **Services** section, and then turn off the toggle for **Save screenshots of site for History**.  All previously saved images will be erased.

Reviewer 3: this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reviewer: ```
Or, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Search and services**, and then turn off the toggle 
for **Save screenshots of site for History**.
```

* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.

> * In Microsoft Edge settings, turn on the toggle **Privacy and security** > **Personalization & advertising** > **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news**.

Reviewer 3: correct.

Reviewer: ```
* In Microsoft Edge, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security** > **Personalization & advertising**, 
and then turn on the toggle **Allow Microsoft to save your browsing activity including 
history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge 
and Microsoft services like ads, search, shopping and news**.
```

Non-enterprise users on Windows devices may also access IE mode.  To turn on IE mode, go to `edge://settings/defaultBrowser` and select the **Allow sites to be reloaded in Internet Explorer mode** setting.  To open tabs in IE mode, open **Settings and more (...)** > **More tools** and select **Reload in Internet Explorer mode**.  After you turn on IE mode, Microsoft Edge periodically requests a list of unsupported sites from a Microsoft service.  The request is sent over HTTPS and does not contain any identifiers.

> Internet Explorer browsing data is stored locally in Microsoft Edge and Internet Explorer.  To delete browsing data while browsing in IE mode, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and clear the data from both **Clear browsing data** and **Clear browsing data for Internet Explorer**.

Reviewer 3: Correct.

Reviewer: ```
To delete browsing data while browsing in IE mode, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then clear the data from both 
the **Clear browsing data** section and the **Clear browsing data for Internet Explorer** section.
```

The jump list in Microsoft Edge lets you easily find your most recently closed sites. Hover on the Microsoft Edge icon in the task bar and right-click. The last three closed tabs are stored locally for each profile.  To delete sites from the jump list in Windows 10 and later, right-click the site, and then select **Remove from this list**.

> You can clear or change the display of your recently closed tabs in the jump list. Go to `edge://settings/privacy`,<!--todo: check, maybe replace by GUI nav path--> and select the **Choose what to clear every time you close the browser** setting. When using an **InPrivate** window, Microsoft Edge does not add closed tab information to the jump list.  When using **Guest** mode, the jump list is not available.  For more information about clearing your browsing data, see [View and delete browser history in Microsoft Edge](https://support.microsoft.com/help/10607).

Reviewer 3: correct.

Reviewer: ```
Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security** > **Choose what to clear every time you close the browser**.
```

If you are signed in to Microsoft Edge with your Microsoft account, you may manage your browsing activity associated with the new tab page from the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/ad-settings).

> Microsoft Edge collects diagnostic data about how you use the new tab page, such as interactions with the search box and selections on quick link tiles.  To enable collection of diagnostic data about how you use the new tab page, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and turn on the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting.  The browser sends diagnostic data about how you use the Microsoft News page to Microsoft to help understand user interactions with news content and improve Microsoft products.  You may turn off Microsoft News content by selecting the **customize** button on the new tab page.  News data is sent to Microsoft using HTTPS and stored for up to 13 months, after which it is aggregated and the personal identifiers are removed.

Reviewer 3: correct.

Reviewer: ```
To enable collection of diagnostic data about how you use the new tab page, 
select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** 
> **Privacy and security**, and then turn on the setting 
**Help improve Microsoft products by sending optional diagnostic data about 
how you use the browser, websites you visit, and crash reports**.
```

Microsoft Edge also supports saving your payment information to your Microsoft account if you are signed in and syncing, which makes the information available across devices. To save payment information to your Microsoft account, credit card verification (CVV) may be required, depending on your current region. CVV is only used for authorization and will not be stored by Microsoft. 

Microsoft Edge supports the PaymentRequest API. The API lets you pay for purchases with payment information you previously saved using autofill. The PaymentRequest API allows the merchant to request the following information: credit card number, credit card expiration, full name, billing address, email address, phone number, and shipping address. The API tells the merchant that you have credit card information saved, but does not share any information with the merchant unless you allow it.  To turn off the Payments feature, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings**, and then in the **Search settings** text box, enter **wallet**.  Click the **Open Wallet** button, and then select **Settings** on the left.

> selkego to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->

Reviewer: deleting line...

Microsoft Edge also supports saving your payment information to your Microsoft account if you are signed in and syncing, which makes the information available across devices. To save payment information to your Microsoft account, credit card verification (CVV) may be required, depending on your current region. CVV is only used for authorization and will not be stored by Microsoft. 

> Microsoft Edge supports the PaymentRequest API. The API lets you pay for purchases with payment information you previously saved using autofill. The PaymentRequest API allows the merchant to request the following information: credit card number, credit card expiration, full name, billing address, email address, phone number, and shipping address. The API tells the merchant that you have credit card information saved, but does not share any information with the merchant unless you allow it. To turn off the Payments feature, go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->

Reviewer 3: I don't believe I've seen this in Settings before. 
I dont see it on either Services or Privacy and security. 

I believe this is supposed to be **edge://wallet/paymentMethods**

Reviewer: and then select p.c.s. - gives: 
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/abc046e7-6527-483d-9e7e-427dd8ebd115)

Reviewer: ```
To turn off the Payments feature, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings**, and then in the **Search settings** text box, enter **wallet**.  
Click the **Open Wallet** button, and then select **Settings** on the left.
```

You may modify your interests or opt-out of personalized ads from the [Ad settings](https://account.microsoft.com/privacy/ad-settings) on the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).

> Opting out of personalized ads on the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/) does not turn off the collection and use of your browsing history for personalization of search results and content in your news feed. You can turn off the collection and use of your Microsoft Edge browsing history for personalized search results and news. Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  In **Personalize your web experience** turn off the **Improve your web experience by allowing Microsoft to use your browsing history from the account for personalizing advertising, search, news and other Microsoft services** setting. If you stop sharing the data, Microsoft no longer collects and uses your browsing history to personalize ads, search results, and news. For more information about personalization in Microsoft Edge, see [Microsoft Edge browsing history for personalized advertising and experiences](https://support.microsoft.com/help/4532583).

Reviewer 3: correct.

Reviewer: ```
Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security** > **Personalize your web experience**, 
and then turn off the setting **Improve your web experience by allowing Microsoft 
to use your browsing history from the account for personalizing advertising, search, 
news and other Microsoft services**.
```

More controls and configurations are available for organizations.  For more information about controls and configurations for organizations, see [Microsoft Edge configurations and experimentation](/deployedge/edge-configuration-and-experiments).

> As a user, you aren't able to turn off the browser updates controlled or configured by your organization. You can control whether your product usage data is sent to Microsoft. Go to `edge://settings/privacy`,<!--todo: check, maybe replace by GUI nav path--> and change the **Optional diagnostic data** settings.

Reviewer 3: correct.

Reviewer: You can control whether your product usage data is sent to Microsoft: select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then change the **Optional diagnostic data** settings.

Microsoft developers need to understand how new functions affect Microsoft Edge and Microsoft services. Microsoft Edge sends a resettable identifier unique to your browser and a functions tag that encodes which new functions were enabled for Microsoft Edge and Microsoft services. New functions help build the best experiences and the best browser for everyone.

> The functions tag is not unique to your installation of Microsoft Edge. The tag is shared across all Microsoft Edge instances that share the same set of new functions.  Microsoft Edge sends the information via HTTPS to Microsoft services. The browser does not send the information when you browse **InPrivate** or in **Guest** mode. You can prevent the data from being sent. Go to `edge://settings/privacy`,<!--todo: check, maybe replace by GUI nav path--> and turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. For more information about how to reset the identifier unique to your browser, see the [Diagnostic data about how you use the browser](#diagnostic-data) section.

Reviewer 3: correct.

Reviewer: ```
Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then turn off the setting 
**Help improve Microsoft products by sending optional diagnostic data 
about how you use the browser, websites you visit, and crash reports**.
```

<!-- ====================================================================== -->
## Resolve navigation errors

> If Microsoft Edge detects SSL connection timeouts, certificate errors, or other network issues that may be caused by a captive portal, it sends a request to `http://edge.microsoft.com/captiveportal/generate_204` and checks the response code. Captive portals include a Wi-Fi network at a hotel or airport. If the request is redirected to another URL, Microsoft Edge opens the URL in a new tab, assuming that it is a sign-in page.  Requests to the captive portal detection page are a stateless service. Requests aren't logged, and cookies aren't sent or saved.  On Windows platforms, Microsoft Edge uses a Windows captive portal service. Otherwise, the Microsoft Edge captive portal service is used. You can turn off the service. Go to `edge://settings/privacy`,<!--todo: check, maybe replace by GUI nav path--> and turn off the **Use a web service to help resolve navigation errors** setting.

Reviewer 3: this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

1. Select **Settings and more** > **Settings**. 

> 1. Select **Privacy and security**. 

Reviewer: Combine two lines per GUI nav path efficient concise convention:
1. Select **Settings and more** > **Settings** > **Privacy and security**. 
or:
1. Select **Settings and more** > **Settings** > **Search and services**. 

The Suggested Change UI is unavailable here.

Reviewer 3: Correct. this is supposed to be Privacy and security

Reviewer: ```
To manage the collection and use of your search results activity for product improvement, 
select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** 
> **Privacy and security**, and then turn on or off the setting for 
**Help improve Microsoft products by sending the results from searches on the web**.
```

By default, your current DNS service provider is used to avoid disruptions to your browsing. Not all service providers offer secure DNS. To avoid delays in browsing, if the secure DNS connection fails, Microsoft Edge attempts a DNS lookup with your current DNS service provider that is not encrypted.

> Microsoft Edge allows you to use a specific secure DNS provider. If a secure DNS provider is selected, then Microsoft Edge does not fall back to regular DNS lookup if the secure lookup fails.  To control your secure DNS settings, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**.

Reviewer: "Privacy and security", or "Search and services"?
@PatrizioBytes 

url was malformed as edge://setting/privacy so missed the global pass

Reviewer 3: it should be Privacy and security end point :)

The shopping service is turned on by default for all users.  To change the shopping setting in Microsoft Edge, complete the following actions.

> 1.  Go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->

Reviewer 3: this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reviewer: ```
To change the shopping setting in Microsoft Edge, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Search and services**, 
and then turn off the **Save time and money with Shopping in Microsoft Edge** setting.
```

Being signed in to Microsoft Edge enables single sign-on. You are automatically signed in to certain websites such as Bing, and other identity-powered experiences such as Sync. If you want to limit automatic sign-in to Microsoft sites such as [Bing](https://bing.com), you may sign out of the browser.

> To sign into specific sites again using your user name and password or clear your cookies, go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  For more information about clearing browsing data, see [View and delete browser history in Microsoft Edge](https://support.microsoft.com/help/10607).

Reviewer 3: correct

Reviewer: ```
To sign into specific sites again using your user name and password or clear your cookies, 
go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**.
```

The SmartScreen service stores data about the reputation checks and builds a database of known malicious URLs and files.  The data is stored on secure Microsoft servers and is used only for Microsoft security services.  The data is never used to identify or target you in any way.  Clearing your browsing cache clears all locally stored SmartScreen URL data.  Clearing your download history removes any locally stored SmartScreen data about file downloads.

> SmartScreen is turned on by default for Microsoft Edge.  To disable SmartScreen, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and under **Security** turn off the **Microsoft Defender SmartScreen** setting.  The setting is the same for all profiles associated with the installation of Microsoft Edge on your device.  The setting is not synced across devices.  The setting applies to InPrivate browsing and Guest mode.  If your device is managed with group policies set by your organization, the setting is reflected in Microsoft Edge.  To view the setting, go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  For more information about SmartScreen, see [SmartScreen: FAQ](https://support.microsoft.com/help/17443).

Reviewer 3: correct

Reviewer: ```
To disable SmartScreen, go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, 
and then turn off the **Microsoft Defender SmartScreen** setting.
```
```
To view the setting, go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security** > **Microsoft Defender SmartScreen**.
```

> Optionally, SmartScreen checks the URLs of files you download to see if any are categorized as potentially unwanted apps.  Blocking potentially unwanted apps helps deliver more productive, performant, and delightful Windows experiences.  The setting is turned off by default and is only available on Windows 10 and later devices.  To enable the feature, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and turn on the **Block potentially unwanted apps** setting.  For more information about how potentially unwanted apps are categorized, see [Potentially unwanted application (PUA)](/windows/security/threat-protection/intelligence/criteria#potentially-unwanted-application-pua).  For more information about how to configure the setting, see [Detect and block potentially unwanted applications](/windows/security/threat-protection/windows-defender-antivirus/detect-block-potentially-unwanted-apps-windows-defender-antivirus).

Reviewer 3: correct

Reviewer: ```
To enable the feature, go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, 
and then turn on the **Block potentially unwanted apps** setting.
```

<!-- ====================================================================== -->
## Suggest similar sites

> To help resolve URL typos in the address bar that result in a website error, Microsoft Edge may recommend a corrected URL.  When a website navigation error occurs, Microsoft Edge sends the domain of the web address to the Microsoft service to suggest a corrected URL.  Microsoft Edge does not include identifiers or tokens with the domain.  If the service finds a suggestion, it returns the suggested URL.  Microsoft stores the incorrect domain, and suggested domain, to help improve the service.  To help you go to the correct sites, the feature is turned on by default.  To turn off the feature, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and under the **Services** turn off the **Suggest similar sites when a website can't be found** setting.

Reviewer 3: This should ready Privacy and security

![The 'Support nonprofits' icon in the address bar](./index-images/support-nonprofit.png)
 

> To control this feature, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and turn on or off the **Show opportunities to support causes and nonprofits you care about** setting.

Reviewer 3: this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reviewer: ```
To control this feature, go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, 
and then turn on or off the **Show opportunities to support causes and nonprofits you care about** setting.
```

All tab information that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time. 

> The Microsoft Edge Tab organization behavior is turned on by default. To turn this behavior off, go to `edge://settings/privacy`<!--todo: check, maybe replace by GUI nav path--> and under **Services** turn off the **Let Microsoft Edge help keep your tabs organized** setting:

Reviewer 3: this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reviewer: ```
The Microsoft Edge Tab organization behavior is turned on by default. 
To turn this behavior off, go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Search and services**, 
and then turn off the **Let Microsoft Edge help keep your tabs organized** setting.

When the Tab Organization service is turned off, new Tab groups are given a generic name 
such as **Group 1**, and you can then rename the Tab group to be more descriptive.  
```
deleted outdated png shown at https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/#tab-organization

<!-- ====================================================================== -->
## Tracking prevention

> Microsoft Edge is designed to detect and block known trackers.  Users may choose from three levels of tracking prevention: Basic, Balanced, and Strict.  To protect user privacy, Balanced is selected by default.  Microsoft Edge detects trackers before any are loaded on the page by using an open-source list of known trackers.  The list is downloaded to the device periodically as the list is updated.  The number of trackers blocked and names of those trackers are stored locally on the device for statistical purposes.  To clear the data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security** > **Blocked trackers**.  The detection and blocking of trackers occurs locally on the device.  To disable tracking prevention, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**.  For more information about Tracking prevention, see [Learn about tracking prevention in Microsoft Edge](https://support.microsoft.com/help/4533959).

Reviewer 3: Privacy and security.

Reviewer: probably Delete this comment, it's in wrong place; in latest Files Changed view, there's no "todo" re: edge://settings/privacy here.  @PatrizioBytes

Author: I'm adding `and then turn off the setting **Tracking prevention**` here, to match the other instances. But, yes, we can resolve this comment.

<!-- ====================================================================== -->
## Tracking prevention

> Microsoft Edge is designed to detect and block known trackers.  Users may choose from three levels of tracking prevention: Basic, Balanced, and Strict.  To protect user privacy, Balanced is selected by default.  Microsoft Edge detects trackers before any are loaded on the page by using an open-source list of known trackers.  The list is downloaded to the device periodically as the list is updated.  The number of trackers blocked and names of those trackers are stored locally on the device for statistical purposes.  To clear the data, go to `edge://settings/privacy/blockedTrackers`.<!--todo: check, maybe replace by GUI nav path-->  The detection and blocking of trackers occurs locally on the device.  To disable tracking prevention, go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  For more information about Tracking prevention, see [Learn about tracking prevention in Microsoft Edge](https://support.microsoft.com/help/4533959).

Reviewer 3: correct. Privacy and security

Reviewer: To clear the data, go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security** > **Blocked trackers**.  The detection and blocking of trackers occurs locally on the device.  To disable tracking prevention, go to **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**.

Microsoft Edge WebView controls allow app developers to host web content in native applications on Windows 7, Windows 10, and later; and selected non-Windows platforms. The applications hosting the WebView2 instance may send diagnostic data with its own identifier to Microsoft. Diagnostic data can include how you use the Microsoft Edge, and sites you visit.

> To enable diagnostic data collection, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**, and then turn on the **Optional diagnostic data** setting. To turn off diagnostic data collection for Microsoft Edge on Windows 10 and later, open **Start** > **Settings** > **Privacy** and select **Diagnostics & feedback**. To turn off diagnostic data collection for all other platforms, go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  Turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. The applications hosting Microsoft Edge WebView may collect other data that is governed by the data collection management of the developer and relevant privacy policies.

Reviewer: is edge://settings/privacy correct?  @PatrizioBytes

Reviewer 3: Correct. This is Privacy and security

Reviewer: ```
For all other platforms, to turn off diagnostic data collection, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Privacy and security**, and then turn off the setting 
**Help improve Microsoft products by sending optional diagnostic data about 
how you use the browser, websites you visit, and crash reports**.
```

Microsoft Edge WebView controls allow app developers to host web content in native applications on Windows 7, Windows 10, and later; and selected non-Windows platforms. The applications hosting the WebView2 instance may send diagnostic data with its own identifier to Microsoft. Diagnostic data can include how you use the Microsoft Edge, and sites you visit.

> To enable diagnostic data collection, go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  Turn on the **Optional diagnostic data** setting. To turn off diagnostic data collection for Microsoft Edge on Windows 10 and later, open **Start** > **Settings** > **Privacy** and select **Diagnostics & feedback**. To turn off diagnostic data collection for all other platforms, go to `edge://settings/privacy`.<!--todo: check, maybe replace by GUI nav path-->  Turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. The applications hosting Microsoft Edge WebView may collect other data that is governed by the data collection management of the developer and relevant privacy policies.

Reviewer 3: Correct. Privacy and security

Reviewer: ```
To enable diagnostic data collection, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy and security**, and then turn on the **Optional diagnostic data** setting.
```
